index.js

```diff
@@ -1,4 +1,4 @@
 'use strict';
-module.exports = function (date) {
-	return (date || new Date()).toISOString().replace(/T/, ' ').replace(/\..+/, ' UTC');
+module.exports = function (options) {
+	return (options && options.date ? options.date : new Date()).toISOString().replace(/T/, ' ').replace(/\..+/, options && options.local === true ? new Date().toString().match(/GMT...../)[0].replace(/GMT/,' UTC') : ' UTC');
```

Comment:
This replaces GMT with UTC and adds a whitespace. Although, in the hindsight, it can be done better and even we can totally give away with UTC as both UTC and GMT are essentially the same thing.